editoast/src/fixtures.rs

```diff
@@ -338,6 +339,57 @@ pub mod tests {
         .await
     }
 
+    pub struct ScenarioV2FixtureSet {
+        pub project: TestFixture<Project>,
+        pub study: TestFixture<Study>,
+        pub scenario: TestFixture<ScenarioModelV2>,
+        pub timetable: TestFixture<TimetableV2>,
+        pub infra: TestFixture<Infra>,
+    }
+
+    #[fixture]
+    pub async fn scenario_v2_fixture_set(
+        db_pool: Data<DbPool>,
+        #[future] timetable_v2: TestFixture<TimetableV2>,
+        #[future] project: TestFixture<Project>,
+    ) -> ScenarioV2FixtureSet {
+        let empty_infra = empty_infra(db_pool.clone()).await;
+        let timetable = timetable_v2.await;
+        let project = project.await;
+        let study = Study::changeset()
+            .name("test_study".into())
+            .description("test".into())
+            .creation_date(Utc::now().naive_utc())
+            .business_code("AAA".into())
+            .service_code("BBB".into())
+            .creation_date(Utc::now().naive_utc())
+            .last_modification(Utc::now().naive_utc())
+            .budget(0)
+            .tags(Tags::default())
+            .state("some_state".into())
+            .study_type("some_type".into())
+            .project_id(project.id());
+        let study: TestFixture<Study> = TestFixture::create(study, db_pool.clone()).await;
```

Comment:
There is a function `study_fixture_set` that exists and which can be used here

Reply:
i have a stackoverflow when i use `study_fixture_set` this is why i created `scenario_v2_fixture_set`

Reply:
i found a solution